.npmignore should not exclude dist

**Describe The Bug:**
When you want to publish your plugin (npm publish) it cannot be installed by other npm users because the dist-directory is excluded from the tar-ball.

**To Reproduce:**
npm publish on dev machine
npm install on user machine

**Expected behavior:**
plugin is correctly installed and can be loaded by homebridge

**Logs:**

```
[11/14/2021, 11:58:39 PM] ERROR LOADING PLUGIN homebridge-xxx:
[11/14/2021, 11:58:39 PM] Error: Cannot find module '/usr/lib/node_modules/homebridge-xxx/dist/index.js'
Require stack:
- /usr/lib/node_modules/homebridge/lib/plugin.js
- /usr/lib/node_modules/homebridge/lib/pluginManager.js
- /usr/lib/node_modules/homebridge/lib/server.js
- /usr/lib/node_modules/homebridge/lib/cli.js
- /usr/lib/node_modules/homebridge/bin/homebridge
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Plugin.load (/usr/lib/node_modules/homebridge/src/plugin.ts:192:73)
    at PluginManager.initializeInstalledPlugins (/usr/lib/node_modules/homebridge/src/pluginManager.ts:131:22)
    at Server.start (/usr/lib/node_modules/homebridge/src/server.ts:156:5)
```

**Plugin Config:**
na

**Screenshots:**
na

**Environment:**

* **Plugin Version**: na
* **Homebridge Version**: na
* **Node.js Version**: v17.0.1
* **NPM Version**: 8.1.0
* **Operating System**: macOS

--- a/.npmignore
+++ b/.npmignore
@@ -101,7 +101,6 @@ web_modules/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
 
 # Gatsby files
 .cache/

